resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/config.scala

```diff
@@ -225,6 +225,15 @@ package object config extends Logging {
     .timeConf(TimeUnit.MILLISECONDS)
     .createWithDefaultString("1s")
 
+  private[spark] val REPORT_LOG_FREQUENCY = {
+    ConfigBuilder("spark.yarn.report.logging.frequency")
+      .doc("Number of application reports processed until the next application status is logged." +
```

Comment:
To be more precise, it is the **maximum** number of reports processed because a state change will cause it to be logged right away. I think the state change behavior is worth mentioning.